alkemio_virtual_contributor_engine/rabbitmq.py

```
@@ -0,0 +1,61 @@
+            self.input_queue_name, auto_delete=False, durable=True
+        )
+        self.result_queue = await self.channel.declare_queue(
+            self.result_queue_name, auto_delete=False, durable=True
+        )
+        self.exchange = await self.channel.declare_exchange(
+            env.rabbitmq_exchange, ExchangeType.DIRECT, durable=True
+        )
+        await self.result_queue.bind(self.exchange, env.rabbitmq_result_routing_key)
+
+    async def consume(self, callback):
+        if not self.input_queue is None:
```

Ruff

Test for object identity should be `is not`

Convert to `is not`

(E714)